d2renderers/d2svg/d2svg.go

```diff
@@ -224,6 +227,39 @@ func arrowheadMarker(isTarget bool, id string, connection d2target.Connection) s
 				width*0.6, height*7/8,
 			)
 		}
+	case d2target.FilledCircleArrowhead:
+		attrs := fmt.Sprintf(`class="connection" fill="%s" stroke-width="%d"`, connection.Stroke, connection.StrokeWidth)
+		offset := 4.0 + float64(connection.StrokeWidth*2)
```

Comment:
why do you need this offset?

Reply:
I was just using the crows feet PR as a reference while out mobile yesterday.  I will try to remove it once I am home in a few hours.

Reply:
Removed the offset. Apologies for the sloppy work there, was doing this on mobile with an ipad yesterday.

Reply:
wow, iPad? 😅